cmd/src/users_clean.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
+	usage := `
+This command removes users from a Sourcegraph instance who have been inactive for 60 or more days.
+	
+Examples:
+
+	$ src users clean -days 182
+	
+	$ src users clean -noAdmin -removeNeverActive 
+`
+
+	flagSet := flag.NewFlagSet("clean", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src users %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		daysToDelete       = flagSet.Int("days", 60, "Days threshold on which to remove users, must be 60 days or greater and defaults to this value ")
+		noAdmin            = flagSet.Bool("noAdmin", false, "Omit admin accounts from cleanup")
```

Comment:
nit: I think giving the opposite option is better, ie. skip admin by default and use the flag `includeAdmin` instead. It feels weird a user clean up ends up deleting admin accounts 🤔

Reply:
This totally makes sense 👍, changed it